system/wine-staging/wine-staging.SlackBuild

```diff
@@ -87,6 +88,15 @@ case "$ARCH" in
   ;;
 esac
 
+MULTILIB=${MULTILIB@L}
```

Comment:
I don't think these conditionals are correct. There is now no way to build on x86_64 but only WINE64.